vocabularies/Common.xml

```diff
@@ -795,10 +795,17 @@ The minimum structure is
         </Property>
       </ComplexType>
       <ComplexType Name="ValueListParameterIn" BaseType="Common.ValueListParameter">
+        <Property Name="Operator" Type="Edm.PrimitiveType" DefaultValue="EQ" Nullable="true">
+          <Annotation Term="Core.Description" String="Operator applied to the property. Defaults to `EQ`." />
+        </Property>
         <Property Name="LocalDataProperty" Type="Edm.PropertyPath" Nullable="false">
-          <Annotation Term="Core.Description" String="Path to property that is used to filter the value list with `eq` comparison" />
+          <Annotation Term="Core.Description" String="Path to property that is used to filter the value list by applying the operator (Single value or lower interval boundary)" />
           <Annotation Term="Core.LongDescription" String="In case the property path contains a collection-based navigation or structural property, the filter is a set of `eq` comparisons connected by `or` operators" />
         </Property>
+        <Property Name="LocalDataPropertyBT" Type="Edm.PropertyPath" Nullable="true">
```

Comment:
As I said before, do not clutter the type with additional properties. Define a new subtype.

Reply:
=> https://github.com/SAP/odata-vocabularies/commit/2e24e353bd0f15f9e82d3adc4244e098f2f2cea1